style.css

```diff
@@ -0,0 +1,253 @@
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  overflow-x: hidden;
+}
+
+/* color classes */
+.nav-bar-color {
+  color: #3a4a42;
+}
+
+.social-icon-color {
+  color: #505f79;
+}
+
+.white-background-color {
+  color: #fff;
+}
+
+.headline-color {
+  color: #172b4d;
+}
+
+.header-color {
+  color: #70c572;
+}
+
+/* Mobile Styles */
+@media screen and (min-width: 368px) {
+  .header {
+    background-image: url(./assets/images/Header-llustration-desktop.png);
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 100vh;
+    position: absolute;
+    top: 92.8px;
+  }
+
+  .header-content {
+    position: relative;
+    width: 100%;
+    max-width: 600px;
+    text-align: center;
+    z-index: 1;
+    align-items: center;
+  }
+
+  .toolbar-menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    position: relative;
+  }
+
+  .nav-bar {
+    position: absolute;
+    top: 0;
+    left: 0;
+    background-color: #fff;
+    color: #3a4a42;
+    width: 100%;
+    height: 100%;
+    text-align: center;
+    padding: 5%;
+    font-size: 8vw;
+    font-weight: 600;
+    line-height: 14vw;
+  }
+
+  .menu li {
+    display: block;
+    margin: 5% 0;
+    list-style: none;
+  }
+
+  .hamburger {
+    cursor: pointer;
+    font-size: 10vw;
+    position: absolute;
+    top: 20px;
+    right: 20px;
+    z-index: 9999;
+  }
+
+  .menu li a {
+    padding: 5%;
+    display: block;
+    font-weight: bold;
+    text-decoration: none;
+  }
+
+  /* Show menu items on hover */
+  .toolbar-menu:hover .nav-bar {
+    display: block;
+  }
+
+  .nav-bar ul {
+    list-style: none;
+    padding-left: 0;
+  }
+
+  /* .nav-bar::after {
+    content: "\f00d";
+    font-family: 'Font Awesome 5 Free', sans-serif;
+    font-weight: 900;
+    font-size: 10vw;
+    position: absolute;
+    bottom: -20px;
+    right: 10%;
+    cursor: pointer;
+    z-index: 2;
+  } */
+
+  .social-icons li {
+    margin-right: 10px;
+  }
+
+  .social-icons li:last-child {
+    margin-right: 0;
+  }
+
+  .chevron-icon {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    padding: 8px;
+    position: absolute;
+    left: calc(50% - 40px / 2 + 0.5px);
+  }
+
+  .social-icons {
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
```

Comment:
- [ ] I would suggest that you properly align the social icons to look exactly like the original design.

> Figma design
<img width="188" alt="Screenshot 2023-04-28 at 13 36 53" src="https://user-images.githubusercontent.com/71140133/235151423-829ac000-f06c-4a62-9f4c-8035aaf9f6d4.png">


> Your design
<img width="377" alt="Screenshot 2023-04-28 at 13 38 15" src="https://user-images.githubusercontent.com/71140133/235151510-85a19c13-9813-4ada-8341-4a093b761298.png">

Reply:
Thank you @Yerimah ,I will make the changes.